File: src/main/java/plugin/command/player/MasterCommandPlugin.java
package plugin.command.player;

import org.redrune.game.entity.actor.player.Player;
import org.redrune.game.content.plugin.type.CommandPlugin;
import org.redrune.utility.constants.SkillConstants;
import plugin.command.CommandManifest;

/**
 * @author Tyluur <itstyluur@gmail.com>
 * @since 9/1/2017
 */
@CommandManifest(description = "Sets your stats to max")
public class MasterCommandPlugin extends CommandPlugin {
	
	@Override
	public void handle(Player player, String[] args, boolean console, boolean clientCommand) {
		for (int skill = 0; skill < 25; skill++) {
			player.getSkills().set(skill, 99);
			player.getSkills().setXp(skill, SkillConstants.getXPForLevel(99));
		}
		player.getSkills().restoreSkills();
	}
	
	@Override
	public String[] identifiers() {
		return arguments("master");
	}
}

File: src/main/kotlin/plugin/command/player/MasterCommandPlugin.kt
package plugin.command.player

import plugin.command.CommandManifest
import org.redrune.game.content.plugin.type.CommandPlugin
import org.redrune.game.global.World
import org.redrune.utility.constants.SkillConstants
import plugin.command.player.YellCommandPlugin
import org.redrune.utility.functions.Misc
import org.redrune.game.entity.actor.player.data.PlayerRight
import org.redrune.utility.constants.InterfaceConstants
import org.redrune.cache.loaders.ItemDefinitions
import org.redrune.game.entity.actor.player.Player

/**
 * @author Tyluur <itstyluur></itstyluur>@icloud.com>
 * @since 9/1/2017
 */
@CommandManifest(description = "Sets your stats to max")
class MasterCommandPlugin : CommandPlugin() {
    override fun handle(player: Player, args: Array<String>, console: Boolean, clientCommand: Boolean) {
        for (skill in 0..24) {
            player.skills[skill] = 99
            player.skills.setXp(skill, SkillConstants.getXPForLevel(99).toDouble())
        }
        player.skills.restoreSkills()
    }

    override fun identifiers(): Array<String> {
        return arguments("master")
    }
}